mage_ai/io/bigquery.py

```diff
@@ -321,8 +321,7 @@ def execute_queries(
             if fetch_query_at_indexes and idx < len(fetch_query_at_indexes) and \
                     fetch_query_at_indexes[idx]:
                 result = result.to_dataframe()
-
-            results.append(result)
+                results.append(result)
```

Comment:
What does it fix?

Reply:
> What does it fix?

It was adding all the results (not DataFrame) from the following execution queries:
```
result = self.client.query(query, **variables)
```
to the returned list. With this code change, it only added the DataFrame results from queries.

Reply:
Actually, when `fetch_query_at_indexes` is None, all the results need to be returned. I'll make a code change to handle this case.

Reply:
I actually don't have context what it's fixing.